src/components/FormComponents/AssetMultiPicker.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import PropTypes from "prop-types";
+import AssetPicker from "./AssetPicker";
+
+export default function AssetMultiPicker({
+  value,
+  onUpdate,
+  stringForm = false,
+  ...props
+}) {
+  return (
+    <>
+      {value.map((asset, index) => (
+        <div
+          key={`${asset === "native" ? "native" : "issued"}${index}`}
```

Comment:
looks we use "native" and "issued" in a couple places, any value to saving them as a const?

Reply:
Actually this key can probably be simplified, this logic is from when I was trying to manage native/not-native via state in the component, and it kept resetting because the key changed.